services/network/wpa_supplicant.py

```diff
@@ -35,7 +35,15 @@ def send_command(self, command):
         """Send a specific command
         """
         print('>', command)
-        self.sock.send(command.encode('utf-8'))
+
+        try:
+            self.sock.send(command.encode('utf-8'))
+        except Exception as e:
+            if self.verbose:
+                print('Exception:', e)
+                print('Failed to send message, reconnecting...')
```

Comment:
does it actually do something to reconnect?

Reply:
My bad, fixed!